docs/en/query_language/functions/rounding_functions.md

```diff
@@ -78,6 +78,92 @@ round(3.55, 1) = 3.6
 round(3.65, 1) = 3.6
 ```
 
+**See Also** 
+
+- [roundBankers](#roundbankers)
+
+## roundBankers(x[, N]) {#roundbankers}
+
+Banker's rounding is a method of rounding fractional numbers, which differs from the usual arithmetic rounding in cases when the rounding number is a half between two numbers, it is rounded to the nearest even number. E.g. 3.5 rounds up to 4, 2.5 rounds down to 2.
+
+Banker's rounding allows to sum rounded numbers without biasing a result.
```

Comment:
Using this method, you can reduce the effect of rounding numbers on the result of summing or subtracting these numbers.